ui/src/features/WorkspaceProjects/hooks.ts

```diff
@@ -122,5 +102,12 @@ export default () => {
     handleDeleteProject,
     handleUpdateValue,
     handleUpdateProject,
+    currentPage,
+    setCurrentPage,
+    totalPages,
+    PROJECTS_FETCH_RATE_PER_PAGE,
```

Comment:
Constants like this should
1) always be defined in the global scope (aka outside of the hook/component,etc function) and in one place.
2) exported and reimported where needed